src/cloudant/design_document.py

```diff
@@ -56,6 +57,17 @@ def views(self):
         """
         return self.get('views')
 
+    @property
+    def search_indexes(self):
+        """
+        Provides an accessor property to the indexes dictionary in the
+        locally cached DesignDocument.
+
+        :returns: Dictionary containing search index names and search functions
+            as key/value
+        """
+        return self.get('indexes')
```

Comment:
How do you guarantee that the `design_document` class is only used for search indexes?
What behaviour are you expecting if I do:
1. `database.create_query_index('myDesignDoc','myIndex' ... )`
2. `database.get_design_document('myDesignDoc').search_indexes`
?

Isn't that is going to return the (query) index called `myIndex` even though I have called a method called `search_indexes`?